schema/fee-waiver.js

```diff
@@ -0,0 +1,57 @@
+const BaseModel = require('./base-model');
+const { uuid } = require('../lib/regex-validation');
+
+class FeeWaiver extends BaseModel {
+  static get tableName() {
+    return 'feeWaivers';
+  }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: ['pilId', 'establishmentId', 'year'],
+      additionalProperties: false,
+      properties: {
+        id: { type: 'string', pattern: uuid.v4 },
+        pilId: { type: 'string', pattern: uuid.v4 },
+        establishmentId: { type: 'integer' },
+        year: { type: 'integer' },
+        comment: { type: 'text' },
+        profileId: { type: 'string', pattern: uuid.v4 },
+        createdAt: { type: 'string', format: 'date-time' },
+        updatedAt: { type: 'string', format: 'date-time' }
```

Comment:
```suggestion
        updatedAt: { type: 'string', format: 'date-time' },
        deleted: { type: ['string', 'null'], format: 'date-time' }
```

Reply:
This doesn't make any difference to anything I don't think because you don't (and shouldn't) ever set the deleted property in an insert or update.

Reply:
I think we normally include it incase we want to add seeds that are deleted (for tests), but we can leave it out until we need it I guess.